### ISSUE TITLE
Soul Frame and others not working as intended

### ISSUE THREAD

**Issue by TomSmash** [GTNewHorizons/GT-New-Horizons-Modpack#13681]
### Your GTNH Discord Username

TomSmash.7157

### Your Pack Version

2.3.2

### Your Server

Single player

### Java Version

Java 20

### Type of Server

None

### Your Expectation

Had 16 apiaries each filled with fertility 4 common drones, bred them with 16 forest drones with each apiary containing 3 soul frames. Didn't get a single purebred cultivated drone or princess. Thought it was just bad luck but the same thing happened with just meadows and forest going for common bees. So I did some more digging.

### The Reality

Went into a creative world. Built an alveary with 16 frame housings. Put soul frames in them (ok 15 soul frames and one oblivion) and still didn't cause winter/forest to mutate to common. Tried metabolic frames and those worked as expected, princess and 4 drones all mutated to purebred commons. Tried acellerated frames and those didn't work. Finally tried those mutation frames from gt with the super low durability. They worked fine in an apiary but the frame housing ate them before the bee got through its life cycle in the alveary.

Also in creative redid the 16x apiaries with 3x frames and confirmed that my results weren't alveary specific.

Something I noticed, the frames should allow you to shift and see their stats when you hover over them. All the frames work like this when they are at full durability but when accelerated frame and soul frame have lost any durability holding shift doesn't reveal any extra data. Metabolic frame and other working frames do.

### Your Proposal

Ensure the frames are working correctly

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by desagas** [comment]
> Something I noticed, the frames should allow you to shift and see their stats when you hover over them. All the frames work like this when they are at full durability but when accelerated frame and soul frame have lost any durability holding shift doesn't reveal any extra data. Metabolic frame and other working frames do.

This is the most annoying thing about doing bees in this pack.

I have found that no set or quantities of frames have ever guaranteed a mutation in this pack. Before I started into HV, I already had a refined bee from RNG, not alvearies. But, now, the only way I know a mutation is almost guaranteed is to use the transmissions and electron tubes with mutation chances. However, the guide for how to use these is vastly inaccurate on their wiki. An alveary with the recommended 2 diamond and 1 diamond tubes, which should work for all bees at all production speeds, according to the wiki, for everything after 2.2.8 will guarantee a pristine to become ignoble, even only having 2 diamond tubes will guarantee the pristine to become ignoble, with no additional frames or other genetic or production modifying things ...

In short, if you are using the guide on the wiki, use it with a grain of salt. Using soul frames has never boosted my mutation chances, so I do not use them anymore.

**Comment by Alastors** [comment]
Would you consider using the Frenzied Frame/Frame of Frenzy from the latest dev version of MagicBees? It has a 10x mutation rate and 0.0001x lifespan for LP

**Comment by Graymare** [comment]
Tested this in 2.2.8 and Soul Frames were similarly broken in both Apiaries and Alvearies - they did not boost mutation chances at all.
